example/test_problem/ELBDM/UniformGranule/python_script/plot_correlation.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import glob
+import re
+import sys
+
+# -------------------------------------------------------------------------------------------------------------------------
+# user-specified parameters
```

Comment:
I prefer to load those parameters with `yt` so they will automatically load-in even with the different initial condition. What do you think? Also, `yt` provides unit conversion, which may be easier for user to know the unit.